### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by Y4k3d0** [henkelmax/replay-voice-chat#40]
### Bug description

The latest version of Replay Voice Chat (1.3.9) causes a crash on startup. I tried launching without it and the game does so flawlessly. After taking a look at the error log, I see the following exception :

```
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'voicechat' at 'de.maxhenkel.voicechat.FabricVoicechatMod'!
```

### Minecraft version

1.21.1

### Simple Voice Chat version

1.21.1-2.5.27

### ReplayMod version

1.21-2.6.19

### Mod version

1.21.1-1.3.9

### Mod loader and version

Fabric Loader 0.16.9

### Steps to reproduce

1. Install both ReplayMod and Replay Voice Chat
2. Launch Minecraft

### Expected behavior

The game starts up with no issue

### Log files

https://gist.github.com/Y4k3d0/f9a8e46864e0ab7d1c01a9ebf25dd446

**Comment by Y4k3d0** [comment]
Oh it's outdated ? I got the latest compatible with 1.21.1. Do I really have to upgrade for it to work ?

**Comment by henkelmax** [comment]
Yes. Only the latest version is supported, as stated in the description.
